stacks/management/overlays/default/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
We don't use the `overlays` convention in this project.
The base should be inside the `modules` dir, and the overlay-like files are in the `registry` or `stacks`.